src/log-imported-targets.ts

```diff
@@ -12,7 +13,8 @@ export function generateImportedTargetData(
   integrationId: string,
   target: Target,
 ): string {
-  return `${orgId}:${integrationId}:${Object.values(target).join(':')}`;
+  const targetData =  _.pick(target, ...targetProps)
```

Comment:
rely on javascript keeping the props in the same order, this can be manually done if proven nessesary